index.bs

```diff
@@ -11391,15 +11392,56 @@ The algorithm for rendering a block of audio from a {{BaseAudioContext}}
 <em>Q</em> is comprised of multiple steps and explained in further detail
 in the algorithm of <a href="#rendering-a-graph">rendering a graph</a>.
 
-<div class="note">
-	In practice, the {{AudioContext}} <a>rendering thread</a> is
-	often running off a system-level audio callback, that executes in
-	an isochronous fashion.
-
-	An {{OfflineAudioContext}} is not required to have a
-	system-level audio callback, but behaves as if it did with the
-	callback happening as soon as the previous callback is
-	finished.
+<div class="correction proposed" id="c2475-1">
+	<span class="marker">Proposed Addition
+		<a href="https://github.com/WebAudio/web-audio-api/issues/2475">Issue
+			2375</a>.
+	</span>
+	Add definition for various concepts related to system-level audio callbacks
+	<div class="amendment-buttons">
+		Buttons here
+	</div>
+	<ins cite=#c2475>
+		The {{AudioContext}} <a>rendering thread</a> is driven by a
+		<dfn>system-level audio callback</dfn>, that is periodically invoked in an
+		isochronous fashion. This <a>system-level audio callback</a> is called
+		repeatedly, and each call has a <dfn>system-level audio callback buffer
+		size</dfn> (that is potentially variable between calls), which is the
```

Comment:
| (that is potentially variable between calls)

Perhaps we can drop this and merge it to the next sentence?